### PR TITLE
feat: Integrate Incentives with CMS

### PR DESCRIPTION
**Pull request by israelswf**
## What's the purpose of this pull request?

This PR aims to integrate `Incentives` with the CMS, by adding its properties.

## How does it work?

In the file `sections.json` we can define all the editable properties for each sections. Also it was necessary to update the componentes `Incentives`, `IncentivesHeader` and `IncentivesFooter` to receive/pass the CMS data properly. 

Final Result in my [developer workspace](https://israel--storeframework.myvtex.com/admin/new-cms/faststore/home/edit/ad2fd81d-a53c-4281-8d01-a4fc2f274db3/):

https://user-images.githubusercontent.com/10616243/162507648-1ffceda9-ed6c-4b59-8ab5-2c0f8f022bf0.mp4

## How to test it?

- Set your workspace to `israel` with the command `vtex use israel`
- Sync your project with the CMS watch `vtex cms sync --watch`
- Open the CMS to edit the data in the [developer workspace](https://israel--storeframework.myvtex.com/admin/new-cms/faststore/home/edit/ad2fd81d-a53c-4281-8d01-a4fc2f274db3/)
- Run your project with `yarn develop` to see the data you've edited in the CMS (P.S.:  Every time I edit things in CMS I had to rerun my local project to get the new data)

## Disclaimer

At the moment, there are some _ghost_ 👻  sections showing up in this dev workspace: 
image

This should be caused possibly by another PR that was synced with the `master` workspace, but this should be ok once everything is merged. 

## References

- Related [Jira Task](https://vtex-dev.atlassian.net/browse/FSSS-261)
- [Integrating Store front with CMS](https://faststore.dev/tutorials/cms/0)
- Base.store [PR](https://github.com/vtex-sites/base.store/pull/474) that syncs Incentive component updates

Special thanks to @lmarinho e @mateuspontes for all the support 🤗 

## Checklist

<em>You may erase this after checking them all ;)</em>

- [x] CHANGELOG entry added
- [x] Update sections.json
- [x] Update Incentives componentes to get/pass CMS data

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ For a full history of changes go to [base.store's CHANGELOG](https://github.com/
 ## [Unreleased]
 
 ### Added
-
+- IncentivesHeader is now integrated with CMS
```
Comment (filipewl) [inline CHANGELOG.md:14]: Just to separate more from the _Changed_ section below.

```suggestion
- IncentivesHeader is now integrated with CMS

```

File: src/components/sections/Incentives/IncentivesHeader.tsx [hunk 2]
```diff
@@ -1,38 +1,20 @@
...
-]
+interface Incentive {
+  icon: string
+  title: string
+  firstLineText: string
+}
+
+interface Props {
+  incentives: Incentive[]
+  classes?: string
```
Comment (filipewl) [inline src/components/sections/Incentives/IncentivesHeader.tsx:14]: Maybe we can remove this one if it's not being used.
```suggestion
```

**Review by filipewl — APPROVED** [review]
LGTM!

**Review by tlgimenes — CHANGES_REQUESTED** [review]
Please, don't do these changes in here. Do these changes on base.store and them rebase it in here. This way we keep changes to a minimum, with the only difference being the actual CMS integration, not really the components.

To avoid re-doing this work, you can also backport it to base.store

However, awesome work you did, for me it's a LGTM 🎉

**Pull request merged**